tests/app/submitter/test_converter.py

```diff
@@ -108,5 +106,44 @@ def test_prepare_answers(self):
             self.assertEquals(EXPECTED_RESPONSE["paradata"], answer_object["paradata"])
             self.assertEquals(EXPECTED_RESPONSE["data"], answer_object["data"])
 
+    def test_zero_answers(self):
```

Comment:
I think this method name implies we're testing a survey with no answers, rather than an answer that is zero. Maybe something like `test_answer_with_zero`